gcc/multiple_target.cc

```diff
@@ -385,9 +386,10 @@ expand_target_clones (struct cgraph_node *node, bool definition)
   before = decl1_v;
   DECL_FUNCTION_VERSIONED (node->decl) = 1;
 
+  char attrs2[num_attrs][5] = {"sve","sve2"};
```

Comment:
is num_attrs a global?
I can't seem to find it within this file.

Instead of manually specifying 5 characters in the variable statement do something like this.
const char* attrs2[] = { "sve","sve2" };

Reply:
No I actually forgot to add
const int num_attrs = 2;